src/plugins/plot/src/configuration/SeriesCollection.js

```diff
@@ -97,6 +97,24 @@ define([
                 filters: filters
             }));
         },
+        remove: function (model, identifier) {
+            var index;
+
+            if (!model) {
+                index = _.findIndex(this.models, function (m) {
+                    return _.isEqual(m.domainObject.identifier, identifier);
```

Comment:
So I was hoping that we could do away with the `identifier` argument altogether. Since we know we are always dealing with domain object models now, could we just use -
```JavaScript
return _.isEqual(m.domainObject.identifier, model.domainObject.identifier);` 
```
And do away with the whole `if` block?

Reply:
@akhenry this is outdated, not sure why its showing up as a change right now